src/main/scala/org/allenai/scholar/metrics/ErrorAnalysis.scala

```diff
@@ -0,0 +1,41 @@
+package org.allenai.scholar.metrics
+
+import java.time.Year
+
+import org.allenai.scholar._
+import org.allenai.scholar.metrics.PrecisionRecall._
+
+case class ErrorAnalysis(
```

Comment:
The output of a metrics calculation.  This same class is returned for all per-paper metrics.